### ISSUE TITLE
compile to .COM/.EXE file?

### ISSUE THREAD

**Issue by aquynh** [programble/tetrasm#2]
this is such a great game. but can you get Makefile to compile this to .COM (or .EXE) file to run in DosBox or DosEmu?

thanks.

**Comment by causal-agent** [comment]
It's programmed as an OS, not to run in DOS. It runs in any x86 emulator.

**Comment by aquynh** [comment]
yes but with some little modification, you can compile it to a .COM file as well. running this in DosBox/DosEmu is way more convenient than in a system emulator.

an example: a .COM file can run in DOS emulator on IOS/Android.

thanks.

**Comment by causal-agent** [comment]
I'd be willing to accept a PR for this.

This looks like it contains useful information: http://nullprogram.com/blog/2014/12/09/

**Comment by aquynh** [comment]
the other game was in a very similar situation with your game, and they just made this possible: 
https://github.com/icebreaker/floppybird/commit/588bc20ad86854083466680d3e6b47fac7cf7577.

**Comment by causal-agent** [comment]
That approach doesn't work here, since Tetrasm assembles object files then links them, rather than `%include` all of the source files.

**Comment by causal-agent** [comment]
Managed to hack around that in [feature/com-build](https://github.com/programble/tetrasm/tree/feature/com-build), but I get a whole slew of errors from DOSBox such as:

```
Illegal read from b1e201c, CS:IP      192:     bd3
```
